packages/wdio-runner/package.json

```diff
@@ -34,6 +34,7 @@
     "@wdio/logger": "6.0.0-alpha.0",
     "@wdio/utils": "6.0.0-alpha.0",
     "deepmerge": "^4.0.0",
+    "expect-webdriverio": "^0.1.3-beta.7",
```

Comment:
If we are going to support this I suggest we update the package to no longer include the word `beta`

Reply:
updated version to 0.2.0 and transferred repo to webdriverio organization. 

@christian-bromann what are the next steps?

Reply:
This PR needs an approval and can then be merged.